### ISSUE TITLE
Do I need the absolute path for the TE BED file?

### ISSUE THREAD

**Issue by olivertam** [bvaldebenitom/SoloTE#11]
Hi,

I've been running `SoloTE` with the following arguments:
```
python scripts/SoloTE_pipeline.py --bam STARsolo/Alignment.sorted.bam --threads 10 --outputdir SoloTE_output/ --teannotation annot/SoloTE_TEannot.bed --outputprefix STARsolo
```
And have encountered the following error:
```
samtools view: Could not read file "annot/SoloTE_TEannot.bed": No such file or directory
[E::hts_open_format] Failed to open file "STARsolo_nogenes_overlappingtes.bam" : No such file or directory
samtools index: failed to open "STARsolo_nogenes_overlappingtes.bam": No such file or directory
[E::hts_open_format_impl] Failed to open file STARsolo_nogenes_overlappingtes.bam
Failed to open BAM file STARsolo_nogenes_overlappingtes.bam
Error: Unable to open file annot/SoloTE_TEannot.bed. Exiting.
[E::hts_open_format] Failed to open file "STARsolo_nogenes_overlappingtes.bam" : No such file or directory
```
I was browsing through the code, and noticed that there is a directory change on [this line](https://github.com/bvaldebenitom/SoloTE/blob/8d2b93156ff929d035acbb862dc4a8e6805f42ce/SoloTE_1.02/SoloTE_pipeline.py#L110). However, I can't see where the `TE_bed` variable is modified to allow for the new relative path.

Is this correct? Does that mean I should provide the absolute path for the TE annotation BED file?

Thanks.

**Comment by bvaldebenitom** [comment]
Hi @olivertam,

Thanks again for the feedback. 

Yes, the absolute path was required up until version 1.04. We corrected this in [release of v1.05](https://github.com/bvaldebenitom/SoloTE/releases/tag/1.05) to allow the use of relative paths.

Please let us know if you have further issues.

**Comment by olivertam** [comment]
Hi @bvaldebenitom,

Thank you for the new release. It is working as advertised.
